tests/dialects/test_duckdb.py

```diff
@@ -1,11 +1,14 @@
+from unittest.mock import patch
+
 from sqlglot import ErrorLevel, UnsupportedError, exp, parse_one, transpile
 from tests.dialects.test_dialect import Validator
 
 
 class TestDuckDB(Validator):
     dialect = "duckdb"
 
-    def test_duckdb(self):
+    @patch("sqlglot.helper.logger")
```

Comment:
why are you patching the logger? isolate the logger override to a with statement

Reply:
looks like you did this for presto, which is not good, that should be cleaned up too

Reply:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertLogs